test/tracer/tracer.c

```diff
@@ -0,0 +1,101 @@
+/*
+ * tracer.c
+ *
+ * RISC-V ISA: RV32I
+ *
+ * Copyright (C) 2017-2023 Tactical Computing Laboratories, LLC
+ * All Rights Reserved
+ * contact@tactcomplabs.com
+ *
+ * See LICENSE in the top level directory for licensing details
+ *
+ */
+
+#include <stdlib.h>
+#include <cstdint>
```

Comment:
For me clang complains about this include, I had to change it to stdint.h

Reply:
If lang==C, it's `<stdwhatever.h>`

If lang==C++, it's `<cstdwhatever>` (without `.h`)